webapp/src/webapp/webclient/aside/main.cljs

```diff
@@ -22,110 +22,126 @@
+      (let [is-connection-section-open? (if (= "open" (.getItem js/localStorage "webclient-connections-section"))
+                                          true
+                                          false)
+            is-runbooks-section-open? (if (= "open" (.getItem js/localStorage "webclient-runbooks-section"))
+                                        true
+                                        false)
+            is-metadata-section-open? (if (= "open" (.getItem js/localStorage "webclient-metadata-section"))
+                                        true
+                                        false)]
+        [:aside {:class "h-full flex flex-col gap-8 p-regular border-l border-gray-600 overflow-auto pb-16"}
+         (when (seq run-connections-list-selected)
+           [:div
+            [:> ui/Disclosure {:defaultOpen true}
+             (fn [params]
+               (r/as-element
+                [:<>
+                 [:> (.-Button ui/Disclosure)
+                  {:as (if (empty? run-connections-list-selected)
+                         "div"
+                         "button")
+                   :class (str "h-8 w-full flex justify-between items-center gap-small "
+                               "text-xs text-gray-400 focus:outline-none focus-visible:ring "
+                               "focus-visible:ring-gray-500 focus-visible:ring-opacity-75")}
+
+
+                  [:div {:class "flex items-center gap-small"}
+                   [:span "Running in"]
+                   (when (> (count run-connections-list-selected) 1)
+                     [:div {:class "flex items-center justify-center rounded-full h-4 w-4 bg-blue-500"}
+                      [:span {:class "text-white text-xxs font-semibold"}
+                       (count run-connections-list-selected)]])]
+
+                  (when-not (empty? run-connections-list-selected)
+                    [:> hero-solid-icon/ChevronDownIcon {:class (str (when (.-open params) "rotate-180 transform ")
+                                                                     "text-white h-5 w-5 shrink-0")
+                                                         :aria-hidden "true"}])]
+
+                 [:> (.-Panel ui/Disclosure) {:className "py-regular"
+                                              :static (if (empty? run-connections-list-selected)
+                                                        true
+                                                        false)}
+                  [connections-running-list/main {:schema-disabled? schema-disabled?
+                                                  :show-tree? show-tree?
+                                                  :run-connections-list-selected run-connections-list-selected}]]]))]])
          [:div
-          [:> ui/Disclosure {:defaultOpen true}
+          [:> ui/Disclosure {:defaultOpen is-connection-section-open?}
            (fn [params]
+             (println (.-open params))
```

Comment:
👀

Reply:
Sorry, it is still in progress. I removed all println
Thx to note.